index.html

```diff
@@ -0,0 +1,349 @@
+        <li class="skil button">bootstrap</li>
+        <li class="skil button">Ruby</li>
+      </ul>
+    </div>
+    <a class="see-project-2, button" href="">See Project</a>
+
+
+    <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <!-- <h2 class="desk-prof-art-dash">Data Dashboard Healthcare</h2> -->
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+        industry's standard</p>
+      <ul class="skils">
+        <li class="button">html</li>
+        <li class="skil button">bootstrap</li>
+        <li class="skil button">Ruby</li>
+      </ul>
+    </div>
+    <a class="see-project-2 button" href="">See Project</a>
+
+    <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <!-- <h2 class="desk-prof-art-web">Website Protfolio </h2> -->
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+        industry's standard</p>
+      <ul class="skils">
+        <li class="button">html</li>
+        <li class="skil button">bootstrap</li>
+        <li class="skil button">Ruby</li>
+      </ul>
+    </div>
+    <a class="see-project-2 button" href="">See Project</a>
+    <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <!-- <h2 class=" desk-prof-art">Profesional Art Printing Data</h2> -->
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+        industry's standard</p>
+      <ul class="skils">
+        <li class="button">html</li>
+        <li class="skil button">bootstrap</li>
+        <li class="skil button">Ruby</li>
+      </ul>
+    </div>
+    <a class="see-project-2 button" href="">See Project</a>
+    <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <!-- <h2 class="desk-prof-art-dash">Data Dashboard Healthcare</h2> -->
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+        industry's standard</p>
+      <ul class="skils">
+        <li class="button">html</li>
+        <li class="skil button">bootstrap</li>
+        <li class="skil button">Ruby</li>
+      </ul>
+    </div>
+    <a class="see-project-2 button" href="">See Project</a>
+    <div class="card-2">
+      <h2 class="prof-art">Profesional Art Printing Data</h2>
+      <!-- <h2 class="desk-prof-art-web">Website Protfolio </h2> -->
+      <p class="p22">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+        industry's standard</p>
+      <ul class="skils">
+        <li class="button">html</li>
+        <li class="skil button">bootstrap</li>
+        <li class="skil button">Ruby</li>
+      </ul>
+    </div>
+    <a class="see-project-2 button" href="">See Project</a>
+
```

Comment:
Nice work on your project section😉 Lets make this look even more like the design😉 take note of the `see project` button😉
yours
<img width="486" alt="card text" src="https://user-images.githubusercontent.com/64643995/220947482-bb28f1b9-6c69-4531-a253-017af944fae7.png">

design
<img width="167" alt="design text" src="https://user-images.githubusercontent.com/64643995/220947600-a0a2451f-592a-47f4-ac2c-2f6cc897f4b2.png">

Kindly take note that the technologies used do not need any hover state, since they are not to be interacted with😉😉